### ISSUE TITLE
Gradient descent

### ISSUE THREAD

**Issue by tiepvupsu** [tiepvupsu/ebookML_src#15]
trang 41: Hai biến ngẫu nhiên x và ycó thể
--> chữ y và có bị dính liền nhau

trang 142: (x2, it2) = myGD1(5, 1)
--> với cùng learning rate η = 0.1, vì sao là myGD1(5,1) mà không là myGD1(5, .1) ???
